src/org/anjocaido/groupmanager/GroupManager.java

```diff
@@ -167,22 +167,10 @@ public void onDisable(boolean restarting) {
 			BukkitPermissions.removeAllAttachments();
 		}
 
-		if (!restarting) {
-
-			if (WorldEvents != null)
-				WorldEvents = null;
-
-			BukkitPermissions = null;
-
-		}
 
 		// log that we are disabled.
 		PluginDescriptionFile pdfFile = this.getDescription();
-		GroupManager.logger.info(String.format(Messages.getString("GroupManager.DISABLED"), pdfFile.getVersion())); //$NON-NLS-1$
-
-
-		if (!restarting)
-			GroupManager.logger.removeHandler(ch);
+		GroupManager.logger.log(Level.INFO, String.format(Messages.getString("GroupManager.DISABLED"), pdfFile.getVersion())); //$NON-NLS-1$

```

Comment:
This is functionally identical to the code it replaces. The only difference is this requires an import.